tw-plugin/macros/remember.tid

```diff
@@ -47,47 +47,57 @@ type: text/vnd.tiddlywiki
 \end
 
 \define remembercz(id, text, mode: "block", reference: "")
-	<$list filter="[[$mode$]match[inline]]" variable=_>
-		<$macrocall $name=twRememberClozeInline id=<<__id__>> text=<<__text__>> reference=<<__reference__>>/>
-	</$list>
-	<$list filter="[[$mode$]!match[inline]]" variable=_>
-		<$macrocall $name=twRememberClozeBlock id=<<__id__>> text=<<__text__>> reference=<<__reference__>>/>
-	</$list>
+    <$macrocall $name=remembercze id=<<__id__>> text=<<__text__>> extra="" mode=<<__mode__>> reference=<<__reference__>>/> 
```

Comment:
Oh, I was expecting to do this differently...rather than creating an entirely new macro, just put the new field at the end of the argument list. Like

```
\define remembercz(id, text, mode: "block", reference: "", extra:"")
```

Then it would be backwards compatible and we wouldn't have to add in an extra stub and change the macro button to insert something new. It's still easy to use without specifying a mode or reference...

```
<<remembercz "This is my {test} cloze." extra:"Extra content goes here.">>
```

Unless you see some reason that doesn't work?

Reply:
I did on this way just to avoid to put the "extra:", but I think it is only trivial detail.  Your proposal looks cleaner.